CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## TBD
+
+### Bug fixes
+
+* Add autoNotify API for internal use by bugsnag-unity
```

Comment:
I don't think we should add anything about this change in the changelog as it's internal API only.